R/utils.R

```diff
@@ -0,0 +1,8 @@
+#' @keywords internal
+unique_features <- function(x) {
```

Comment:
Is this function ever used?

Reply:
https://github.com/NorskRegnesentral/shapr/blob/6c4d6463f75a2d35488d49c0842df2724c4cfcbd/R/models.R#L296